lib/net.js

```diff
@@ -1524,6 +1524,13 @@ Server.prototype.listen = function(...args) {
     backlog = options.backlog || backlogFromArgs;
     listenInCluster(this, pipeName, -1, -1,
                     backlog, undefined, options.exclusive);
+    let mode = 0;
+    if (options.readableAll === true)
+      mode |= PipeConstants.READABLE_ALL;
+    if (options.writableAll === true)
```

Comment:
Just curious, what is the reason you went for strict true-ness instead of truthiness?